worknenjoy/gitpay
# fix(assign):email information removal of assigned user

## Description
This PR removes Email information for assigned user from following emails contents and both locale
Having subjects
1.Someone is interested in a task that you created on Gitpay
2. You have assigned {{name}} to do a task
## Changes
modules/mail/assign.js
locales/br.json
locales/en.json
locales/result/br.json
locales/result/en.json

## Issue

> Closes #530 

## Impacted Area

> AssignMail.owner.interested
> AssignMail.owner.assigned

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

-create an user
-Import an issue
-Create another user
-Apply to solve an issue in I'm interested (An email should be sent with subject 'Someone is interested in a task that you created on Gitpay')
-Log with the first user
-Assign user
-Accept from Second user (An email should be sent with subject 'You have assigned {{name}} to do a task')
